File: SecondYear/Shubh-dev-19_ShubhKumar_2226cs1054_2/Week1/pattern6.cpp
//{ Driver Code Starts
#include <iostream>

using namespace std;

// } Driver Code Ends
class Solution {
  public:
    void printTriangle(int n) {
        // code here
         for (int i = 1; i <= n; i++) {
       
        for (int j = 1; j <= n - i; j++) {
            cout << " ";
        }

     
        for (int j = 1; j <= 2 * i - 1; j++) {
            cout << "*";
        }

       
        cout << endl;
    }
    }
};

File: SecondYear/Geek-shreya_ShreyaGarg_2226cse1095_2/Week_1/Pattern_6.py
#User function Template for python3

class Solution:
    def printTriangle(self, N):
        k=0
        for i in range(1,N+1):
            for j in range(1,(N-i)+1):
                print(end = " ")
            while k!=(2*i-1):
                print("*", end="")
                k += 1
   
            k = 0
            print()


#{ 
 # Driver Code Starts
#Initial Template for Python 3

if __name__ == '__main__':
    t = int(input())
    for _ in range(t):
        N = int(input())
        ob = Solution()
        ob.printTriangle(N)
# } Driver Code Ends